### PR TITLE
New CheckCountByDateAggType format for Well Architected

### PR DESCRIPTION
**Pull request by BlondJP**

### CODE REVIEW

File: src/wellArchitected/README.md [hunk 1]
```diff
@@ -27,12 +27,14 @@ const query: GetPartnerDataQuery = {
         checksAgg: {
             checks: {
                 data: {
-                    count: true,
```
Comment (BlondJP) [inline src/wellArchitected/README.md:30]: Warning: Breaking change, the delete of count: true can make some problem if some projects use it.

File: src/wellArchitected/types/wellArchitectedGraphQLTypes.ts [hunk 2]
```diff
@@ -117,10 +117,11 @@ export type SeveritiesAggType = {
  */
 export type CheckCountByDateAggType = {
   accounts?: number;
-  count?: number;
```
Comment (BlondJP) [inline src/wellArchitected/types/wellArchitectedGraphQLTypes.ts:120]: risk of breaking change

File: tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts [hunk 3]
```diff
@@ -20,12 +20,14 @@ export const GET_PARTNER_DATA_QUERY: GetPartnerDataQuery = {
     checksAgg: {
       checks: {
         data: {
-          count: true,
```
Comment (BlondJP) [inline tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts:23]: same

File: tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts [hunk 4]
```diff
@@ -20,12 +20,14 @@ export const GET_PARTNER_DATA_QUERY: GetPartnerDataQuery = {
...
           date: true,
+          score: true,
+          scoreUnit: true,
         },
         last: {
-          count: true,
```
Comment (BlondJP) [inline tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts:27]: same

File: tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts [hunk 5]
```diff
@@ -225,7 +227,7 @@ export const GET_PARTNER_DATA_QUERY: GetPartnerDataQuery = {
...
... name progression reference } unregisteredCustomers { name reference offers { iaas { name sku subscriptions { friendlyName partnerRef vendorSubscriptionId } } saas { name subscription } } } } marketplacesAgg { marketplaces { data { date score scoreUnit } last { date score scoreUnit } name partners { data { date failed passed score scoreUnit total } last { date failed passed score scoreUnit total } name progression reference } progression } } monthlyTrendAgg { period { from to } scores { date score scoreUnit severities { name value } } } pagination { currentPage perPage previous next total totalPage } period { from to } results { account { failed name passed reference score standards { currentScore checks { description flagged group isFailed name processed reference score severity } failed maxScore passed reference score total } total } registration { accountReference customer { name reference } marketplace reseller { name reference } subscription { reference } vendorCode } } scoresAgg { last { date failed passed score scoreUnit total } scores { date failed passed score scoreUnit total } } severitiesAgg { severities { data { date value } last { date value } name progression } } }}';
```
Comment (BlondJP) [inline tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts:228]: maybe one here too

File: tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts [hunk 6]
```diff
@@ -237,12 +239,14 @@ export const GET_ADMIN_DATA_QUERY: GetAdminDataQuery = {
     checksAgg: {
       checks: {
         data: {
-          count: true,
```
Comment (BlondJP) [inline tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts:240]: same

File: tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts [hunk 7]
```diff
@@ -649,12 +655,14 @@ export const GET_CUSTOMER_ACCOUNT_DATA_QUERY: GetCustomerAccountDataQuery = {
...
```
Comment (BlondJP) [inline tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts:652]: same

File: tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts [hunk 8]
```diff
@@ -824,4 +832,4 @@ export const GET_CUSTOMER_ACCOUNT_DATA_QUERY: GetCustomerAccountDataQuery = {
...
...t date } last { count date } name progression reference vendorCode } } monthlyTrendAgg { period { from to } scores { date score scoreUnit severities { name value } } } pagination { currentPage perPage previous next total totalPage } period { from to } results { account { failed name passed reference score standards { currentScore checks { description flagged group isFailed name processed reference score severity } failed maxScore name passed reference score total } total } registration { accountReference customer { name reference } marketplace reseller { name reference } subscription { reference } vendorCode } } scoresAgg { last { date failed passed score scoreUnit total } scores { date failed passed score scoreUnit total } } severitiesAgg { severities { data { date value } last { date value } name progression } } standardsAgg { standards { data { date score scoreUnit failed passed total } last { date score scoreUnit failed passed total } name progression reference } } standards { checks { data { date flagged isFailed name processed score scoreUnit } description last { date flagged isFailed name processed score scoreUnit } name progression reference severity } name reference } }}';
```
Comment (BlondJP) [inline tests/wellArchitected/mocks/wellArchitectedQueries.mocks.ts:827]: be careful of the count here

**Review by fdianoux — APPROVED** [review]

**Pull request merged**